src/themes/uswitch/theme.json

```diff
@@ -3954,6 +3924,60 @@
         }
       },
       "variants": {
+        "borderBottomImageLeft": {
+          "wrapper": {
+            "pb": [74, 100],
+            "px": 0
+          },
+          "inner": {
+            "flexDirection": ["column", "column", "row"],
+            "alignItems": ["center", "center", "flex-start"],
+            "px": 0,
+            "borderBottom": [null, null, "1px solid #C4C4C4"]
+          },
+          "colorBlock": {
+            "display": "none"
+          },
+          "image-container": {
+            "alignSelf": ["auto", "flex-end"],
+            "mt": [-40, 0, 20],
+            "px": 0,
+            "pt": [45, 0],
+            "height": [null, "100%"],
+            "mb": [35, 45, 0],
+            "pb": [0],
+            "borderBottom": ["1px solid #C4C4C4", "1px solid #C4C4C4", "none"]
+          },
+          "content": {
+            "alignSelf": "center",
+            "px": [20, 0, 0],
+            "textAlign": "left",
+            "maxWidth": ["100%", null, "grid.md.8"],
+            "width": "100%",
+            "flexBasis": [null, "45%"],
+            "h4": {
+              "width": "100%"
+            },
+            "h2": {
```

Comment:
these sorts of selector are going to make things quite brittle e.g. if we change a h4 to a h3

Reply:
These styles are targeting elements in part of the content block from the rich text editor in Contentful, so we would expect not to have prior knowledge of any class names, and for users to be selecting heading styles as appropriate for their design. Therefore I think it's best to leave as is.

The styles in question were copied over from other left-leaning variants. I considered adding these to the base variant, however, it's probably best not to move these to the base variation as they would need to be nullified for right-leaning variants.